app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java

```diff
@@ -26,10 +26,16 @@
 
 import com.bumptech.glide.Glide;
 import com.framgia.englishconversation.R;
+import com.framgia.englishconversation.data.model.UserModel;
+import com.framgia.englishconversation.data.source.callback.DataCallback;
+
+import com.framgia.englishconversation.data.source.remote.auth.AuthenicationRemoteDataSource;
+import com.framgia.englishconversation.data.source.remote.auth.AuthenicationRepository;
 import com.framgia.englishconversation.utils.Blocker;
 import com.framgia.englishconversation.utils.Constant;
 import com.google.android.exoplayer2.SimpleExoPlayer;
 import com.google.android.exoplayer2.ui.SimpleExoPlayerView;
+import com.google.firebase.auth.FirebaseUser;
```

Comment:
ERROR: Unused import: com.google.firebase.auth.FirebaseUser. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports